TrafficCapture/trafficCaptureProxyServer/README.md

```diff
@@ -0,0 +1,100 @@
+
+1. **Log in to one of the coordinator nodes** for command line access.
+2. **Update node’s port setting**.
+    1. **Update elasticsearch.yml/opensearch.yml**. Add this line to the node’s config file:
+       http.port: 19200
+3. **Restart Elasticsearch/OpenSearch process** so that the process will bind to the newly configured port. For example, if systemctl is available on your linux distribution you can run the following (Note: depending on your installation of Elasticsearch, these methods may not work for you)
+   1. `sudo systemctl restart elasticsearch.service`
+   
+4. **Verify process is bound to new port**. Run netstat -tapn to see if the new port is being listened on.
+   If the new port is not there, then there is a chance that Elasticsearch/ OpenSearch is not running, in that case, you must start the process again. (Depending on your setup, restarting/starting the Elasticsearch process may differ)
+5. **Test the new port** by sending any kind of traffic or request, e.g; curl https://localhost:19200 or http://
+6. **Download Capture Proxy**:
+    1. Go to the Opensearch Migrations latest releases page: https://github.com/opensearch-project/opensearch-migrations/releases/latest
+    2. Copy the link for the Capture Proxy tar file, mind your instance’s architecture.
+    3. `curl -L0 <capture-proxy-tar-file-link> --output CaptureProxyX64.tar.gz`
+    4.  Unpack solution tarball: `tar -xvf CaptureProxyX64.tar.gz`
+    5. `cd CaptureProxyX64/bin`
+7. **Running the Capture Proxy**:
+        1. `nohup ./CaptureProxyX64 --kafkaConnection <msk-endpoint> --destinationUri http://localhost:19200 —listenPort 9200 —enableMSKAuth --insecureDestination &`
+
+    **Explanation of parameters** in the command above:
+    
+    * **--kafkaConnection**: your MSK client endpoint.
+    * **--destinationUri**: URI of the server that the Capture Proxy is capturing traffic for.
+    * **--listenPort**: Exposed port for clients to connect to this proxy. (The original port that the node was listening to)
+    * **--enableMSKAuth**: Enables SASL Kafka properties required for connecting to MSK with IAM auth.
+    * **--insecureDestination**: Do not check the destination server’s certificate.
+
+
+
+8. **Test the port** that the Capture Proxy is now listening to.
+    1. `curl https://localhost:9200` or `http://`
+    2. You should expect the same response when sending a request to either ports (9200, 19200), except that the traffic sent to the port that the Capture Proxy is listening to, will be captured and sent to your MSK Client, also forwarded to the new Elasticsearch port.
+9. **Verify requests are sent to Kafka**
+   * **Verify that a new topic has been created**
+     1. Log in to the Migration Console container.
+     2. Go the Kafka tools directory
+       cd kafka-tools/kafka/bin
+     3. Create the following file to allow communication with an AWS MSK cluster. Name it: `msk-iam-auth.properties`
```

Comment:
Don't we already have a version of this installed as a helper on the migration console?

Reply:
Good catch. I had created that file while testing and forgotten it's already created in one of the directories outside kafka.